### ISSUE TITLE
Nested enum with inline comments break parser

### ISSUE THREAD

**Issue by sevennt** [uber/prototool#501]
Protobuf example:
```protobuf
// FooResponse is my response
message FooResponse {
  enum BarEnum // BarEnum is my nested enum
  {
      BAR_TYPE_INVALID= 0;
      BAR_TYPE_BAD = 1;
  }
  string msg = 1;
}
```
parsing failed.

`emicklei/proto` is already fixed, https://github.com/emicklei/proto/issues/118.

Maybe we can update `emicklei/proto` module ?

**Comment by smaye81** [comment]
Deps were updated with https://github.com/uber/prototool/pull/513.  Can you try to install the `dev` branch and verify whether the upgraded dep fixes your issue?

**Comment by smaye81** [comment]
Closing this as it should be fixed with the 1.9 release.  If this is still an issue, please reopen.
